recipes/opencv/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "2.4.11" %}
+{% set filename = "opencv-%s.zip" % version %}
+
+package:
+  name: opencv
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}
+  url: http://sourceforge.net/projects/opencvlibrary/files/opencv-unix/{{ version }}/{{ filename }}/download
+  sha1: d6e3048416d42213c204f89b9dfe39742f9a708c
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [not py27]
+
+requirements:
+  build:
+   - cmake
+   - numpy x.x
+   - python
+   - eigen 3.*
+   - tbb 4.*  # [osx]
+
+  run:
+   - python
+   - numpy x.x
+   - tbb 4.*  # [osx]
+
+test:
+  imports:
+    - cv2
+    - cv2.cv
+
+about:
+  home: http://opencv.org/
+  license: BSD 3-clause
+  summary: Computer vision and machine learning software library.
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - patricksnape
+
```

Comment:
Yes, though this has two blank lines. If there are no blank lines at the end, `git` will point this out to you and GitHub will put a nasty :no_entry_sign:. If there is only one, both `git` and GitHub won't show them. Again this is super minor.